include/maths/CBoostedTree.h

```diff
@@ -159,12 +161,17 @@ class MATHS_EXPORT CBoostedTree final : public CDataFrameRegressionModel {
     //! Get the column containing the model's prediction for the dependent variable.
     std::size_t columnHoldingPrediction(std::size_t numberColumns) const override;
 
+    //! Estimate the maximum booking memory that training the boosted tree on a data
+    //! frame with \p numberRows row and \p numberColumns columns will use.
+    std::size_t estimateMemoryUsage(std::size_t numberRows, std::size_t numberColumns) const;
```

Comment:
This is on the feature/regression branch. Is this from an earlier revision?

Reply:
This is related to #538 isn't it?

Reply:
I think the confusion is caused by not branching not off `feature/regression` so we're missing some of the committed changes in this branch. Maybe you can completely clear this up with a rebase, or maybe it will be easier to reapply to a branch of `feature/regression`.

Reply:
Indeed I recreated all changes from HEAD of `feature/regression` in the local workspace. I am about to create a PR in the next couple of hours :)

Reply:
See PR #555